desc/compute/_basis_vectors.py

```diff
@@ -556,6 +556,32 @@ def _e_sub_zeta_tz(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="e_theta_PEST",
+    label="\\mathbf{e}_{\\theta_{PEST}}",
+    units="m",
+    units_long="meters",
+    description="Covariant straight field line poloidal basis vector",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=[
+        "0",
+        "R_t",
+        "Z_t",
+        "lambda_t",
+    ],
+)
+def _e_sub_theta_pest(params, transforms, profiles, data, **kwargs):
+    dtdv = 1 / (1 + data["lambda_t"])
```

Comment:
don't need to change but i think `dt_dv` is more clear here

Reply:
I agree with Kaya. If R_t = partial R/partial t, then partial theta/partial v should be dt_dv